viewer/deps.edn

```diff
@@ -0,0 +1,11 @@
+{:paths ["src"]
```

Comment:
Do we need to also add `"../src"` as path here (is that even supported?)? Otherwise we'd be missing the sources. I'd prefer if we could avoid having to move the source files to a different directory.

Reply:
I believe I got an error or warning when I tried adding a src path outside the directory (that was my first approach)